rib/rib_test.go

```diff
@@ -2712,3 +2956,144 @@ func mustChainRIB() *RIB {
 	}
 	return r
 }
+
+// mustTwoEntryRIB returns a RIB that has two IPv4 entries referring to the same -> NHG -> NH, and panics if it cannot.
+// The NHG and NHs are created in the network instance specified by nhNI.
+func mustTwoEntryRIB(nhNI string) *RIB {
+	r := New(defName)
+
+	if nhNI != defName {
+		if err := r.AddNetworkInstance(nhNI); err != nil {
+			panic(fmt.Sprintf("cannot create network instance %s, %v", nhNI, err))
+		}
+	}
+
+	for _, idx := range []uint64{1, 2} {
+		if _, _, err := r.AddEntry(nhNI, &spb.AFTOperation{
+			Id: idx + 2,
+			Entry: &spb.AFTOperation_NextHop{
+				NextHop: &aftpb.Afts_NextHopKey{
+					Index:   idx,
+					NextHop: &aftpb.Afts_NextHop{},
+				},
+			},
+		}); err != nil {
+			panic(fmt.Sprintf("cannot build testcase, can't add nh, got err: %v", err))
```

Comment:
same comment as before about panics. Might want to pass through "t" and use t.Fatal instead.